src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -179,6 +179,18 @@ private IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundInterpolationOperation((BoundStringInsert)boundNode);
                 case BoundKind.LocalFunctionStatement:
                     return CreateBoundLocalFunctionStatementOperation((BoundLocalFunctionStatement)boundNode);
+                case BoundKind.ConstantPattern:
+                    return CreateBoundConstantPatternOperation((BoundConstantPattern)boundNode);
+                case BoundKind.DeclarationPattern:
+                    return CreateBoundDeclarationPatternOperation((BoundDeclarationPattern)boundNode);
+                case BoundKind.WildcardPattern:
+                    return null;
```

Comment:
Yes, see this test: https://github.com/dotnet/roslyn/pull/20276/files#diff-7d83c86cd85172860c4416b70dc793eaR450. The default label here is bound as a BoundWildCardPattern.